src/js/lib/youtube.js

```diff
@@ -21,6 +21,15 @@ export function pimpYouTubePlayer(videoId, node, height, width) {
             function playerReady() {
                 youTubePlayer.playVideo();
             }
+
+            var chapterBtns = [].slice.call(document.querySelectorAll('.docs--chapters li'));
+
+            chapterBtns.forEach( function(chapterBtn){
+              chapterBtn.onclick = function(){
+                var chapTime = parseInt(chapterBtn.getAttribute('data-sheet-attribute'));
```

Comment:
`data-sheet-attribute` is a bit of an odd name, would `data-chapter-timestamp` be more explicit?